### DISCUSSION TITLE
[blocks]: Landing pages

### DISCUSSION THREAD

**Discussion by mohammedzamakhan** [shadcn-ui/ui#3170] [Blocks Request]
### Description

A set of different landing pages, some for tech startups, for services, agencies etc, with different hero sections, banner sections, testimonial sections, email subscription blocks, headers, footers.

### Example

https://tailwindui.com

**Comment by Masstronaut** [comment]
Definitely would love to see this kind of stuff. Given the "blocks" direction, maybe it would make sense to focus most of the effort on making the blocks (think: hero sections, testimonials, price/plan compare table, etc.) and a smaller number of full page buildouts using those blocks.

**Comment by InsightfulFuture** [comment]
+1

**Comment by lea19019** [comment]
Personally I find the Nuxt UI Pro example really nice
https://saas-template.nuxt.dev/

Maybe something like that but with Shadcn would be great!

**Comment by codescandy** [comment]
Like Saas landing pages. 
https://block.codescandy.com/

**Comment by lgs** [comment]
Absolutely, this will boost shadcn-ui adoption even widely outside the current circle.

**Comment by Natnael-Tilahun** [comment]
Came here to request this. Would love to see that.
